src/main/java/it/gov/pagopa/payhub/auth/mypivot/service/MyPivotOperatorsService.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.payhub.auth.mypivot.service;
+
+import it.gov.pagopa.payhub.auth.mypivot.model.MyPivotOperator;
+import it.gov.pagopa.payhub.auth.mypivot.repository.MyPivotOperatorsRepository;
+import java.util.Optional;
+import java.util.Set;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MyPivotOperatorsService {
+
+  private final MyPivotOperatorsRepository myPivotOperatorsRepository;
+
+  public MyPivotOperatorsService(MyPivotOperatorsRepository myPivotOperatorsRepository) {
+    this.myPivotOperatorsRepository = myPivotOperatorsRepository;
+  }
+
+  public void registerMyPivotOperator(String mappedExternalUserId, String organizationIpaCode, Set<String> roles) {
+    Optional<MyPivotOperator> existingMyPivotOperator = Optional.ofNullable(
```

Comment:
again, you could wrap the repository method return into the Optional instead of wrapping it explicitly later